docs/0132.md

```diff
@@ -0,0 +1,167 @@
+# Rules for AEP-132: List method
+
+## List methods: No HTTP body
+
+This rule enforces that all `List` operations omit the HTTP `body`, as mandated
+in [AEP-132][].
+
+### Details
+
+This rule looks for "get" operations on a path that does not end in a path
+parameter, and complains if the 'requestBody' field is present.
+
+### Examples
+
+**Incorrect** code for this rule:
+
+```yaml
+paths:
+  '/books':
+    get:
+      summary: List books
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              type: string
+      responses:
+```
+
+**Correct** code for this rule:
+
+```yaml
+paths:
+  '/books':
+    get:
+      summary: List books
+      responses:
+```
+
+### Disabling
+
+**Important:** HTTP `GET` requests are unable to have an HTTP body, due to the
+nature of the protocol. The only valid way to include a body is to also use a
+different HTTP method (as depicted above).
+
+[aep-132]: https://aep.dev/132
+
+## List methods: Request param types
+
+This rule enforces that all `List` standard methods use the correct type for
+any optional parameters described in [AEP-132][].
+
+## Details
```

Comment:
```suggestion
### Details
```

It looks like it's a sub-section of the `request param types` rule - so should it (and other sections below) be H3 / ###?